hosts.go

```diff
@@ -4,39 +4,49 @@ import (
 	"encoding/json"
 	"net/http"
 
+	"github.com/NebulousLabs/Sia/api"
 	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
 )
 
-// Make new struct with more fields than what will be returned
-type hostList struct {
-	Hosts  []modules.HostSettings
-	Length int
+// hostDisplayInfo contains the data about each host that will be displayed on
+// the hosts page.
+type hostDisplayInfo struct {
+	IPAddress    modules.NetAddress
+	TotalStorage int64
+	Price        types.Currency
+}
```

Comment:
what is the purpose of this type? Just to restrict the available information? Doesn't seem worth the hassle.